### ISSUE TITLE
UAC 2.0.0 Memory Capture didn't run

### ISSUE THREAD

**Issue by anthony-pierce** [tclahr/uac#27]
Hello tclahr,

I attempted to use the option on an Ubuntu 20.04.3 LTS via WSL2 :
sudo ./uac -p full-with-memory-dump /tmp

But a memory file was not generated nor was it logged that it was ran. It did appear in the list of artifacts it was collecting:
[144/182] memory_dump/avml.yaml

There was no record of it in the UAC.log file. I can try it again next week, but wanted to make you aware of it in advance.

I tried again by manually selecting all the artifacts and this did run the memory capture:
sudo ./uac -a memory_dump/\*,live_response/\*,bodyfile/\*,files/\*,chkrootkit/\*,hash_executables/\* /tmp

There was a total of 183 artifacts, but did receive an error in the output tar file, within memory_dump/avml.raw.stderr:
__Error: unable to collect memory

Caused by:
    0: parsing /proc/iomem failed
    1: unable to open file: /proc/iomem
    2: No such file or directory (os error 2)_
_

**Comment by tclahr** [comment]
Hi anthony-pierce,

I was able to run it on an Ubuntu 20.04 TLS via WSL2. I used:
`sudo ./uac -p full-with-memory-dump .`

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.3 LTS
Release:        20.04
Codename:       focal

$ uname -a
Linux tclahr-t14 5.10.16.3-microsoft-standard-WSL2 #1 SMP Fri Apr 2 22:23:49 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

Can you try to run avml alone using the following command? From uac-2.0.0 directory:
`sudo ./tools/avml/bin/linux/x86_64/avml /tmp/test.raw`

**Comment by anthony-pierce** [comment]
Hello tclahr,

I tried running avml as you suggested.

I see the issue must be on my end or specifically with AVML. I tried it on both a Linux server running on a KVM (Ubuntu 20.04.3 LTS) which couldn't acquire physical memory along with trying WSL again and still no luck. It did work fine on an Ubuntu 21.04 as a virtual machine.

KVM server
/dev/crash failed: unable to create image. source:/dev/crash destination:/tmp/test.raw

Caused by:
    0: unable to open source file: /dev/crash
    1: No such file or directory (os error 2)
/proc/kcore failed: No such file or directory (os error 2)
/dev/mem failed: unable to write block: 1048576:3221053439

Caused by:
    0: unable to copy block: 1048576..3221053439
    1: unable to copy block: 1048576..3221053439
    2: Operation not permitted (os error 1)
Error: unable to collect memory

Caused by:
    unable to read physical memory

WSL:
ubuntu:~/uac-2.0.0/tools/avml/bin/linux/x86_64$ sudo ./avml /tmp/test.raw
Error: unable to collect memory

Caused by:
    0: parsing /proc/iomem failed
    1: unable to open file: /proc/iomem
    2: No such file or directory (os error 2)
